### DISCUSSION TITLE
new user and Cronicle vs Orchestra

### DISCUSSION THREAD

**Discussion by claudiobosticco** [jhuckaby/Cronicle#450] [General]
Hello, In the past month I've tested Cronicle, and now I was in the process of starting pushing it in production.
But then I saw the Orchestra announcement. 
Without knowing nothing about this transition the Cronicle proposal may be rejected by my company and I don't want that happen :-(
So I'm trying to understand what will happen when Orchestra will be out:
- Will I be able to migrate the jobs from Cronicle to Orchestra? How effortlessly?
- Will Orchestra be free or not? Can I expect something like the Cronicle features for free and more features with some subscription plan (that may also be interesting)?
- Do you have a roadmap for Orchestra? At least an ETA?

Thank you in advance 
Claudio

**Comment by jhuckaby** [comment]
Hello there!

> Will I be able to migrate the jobs from Cronicle to Orchestra? How effortlessly?

Absolutely!  Orchestra will automatically detect an existing Cronicle installation on your server and prompt you to migrate over your data.  I'll make sure it is easy and seamless.

> Will Orchestra be free or not? Can I expect something like the Cronicle features for free and more features with some subscription plan (that may also be interesting)?

I haven't announced anything yet, but you're on the right track here.  It will definitely be free and open source, but there MAY also be a paid option for additional features (beyond what Cronicle currently offers).

The free, open source Orchestra will have feature parity with Cronicle.  In fact, the entire thing may be free & open source -- I honestly haven't really decided on that yet.

> Do you have a roadmap for Orchestra? At least an ETA?

I apologize for lack of a roadmap.  Orchestra (and also Cronicle) are just hobby projects of mine, and I am a one-man shop.  I work on them in my spare time only.

My initial projection for completion of Orchestra was early 2021, but we're way past that now, sigh.  My new estimate is early 2022.

Meanwhile, I will continue to apply bug fixes and security updates to Cronicle.  In fact, this will continue long after Orchestra is released, as many people may want to stick with Cronicle, and I'm not going to abandon it.

**Reply by claudiobosticco** [reply]
Hello, and thanks a lot for your reply :-)
Now we can continue on the path of adopting Cronicle...keep up the good work!

**Reply by mannharleen** [reply]
Thank you. Eagerly waiting for Orchestra. Until then Cronicle it is

**Reply by LtCMDstone** [reply]
Hey @jhuckaby any news about Orchestra? :-)

**Reply by jhuckaby** [reply]
Hi there!  No real news, other than I'm working on it very hard every day.  The UI is mostly done at this point, but I still have a lot of back-end work to do, not to mention testing and docs.  It's coming along nicely, but it's still several months out.  I will not rest until it is done!
